src/cmdstan-guide/pathfinder_intro.Rmd

```diff
@@ -0,0 +1,112 @@
+# Variational Inference using Pathfinder {#pathfinder-intro}
+
+The CmdStan method `pathfinder` uses the Pathfinder algorithm of @zhang_pathfinder:2022.
+Pathfinder is a variational method for approximately
+sampling from differentiable log densities. Starting from a random
+initialization, Pathfinder locates normal approximations to the target
+density along a quasi-Newton optimization path, with local covariance
+estimated using the inverse Hessian estimates produced by the
```

Comment:
It's negative inverse Hessian.  I think I wrote that wrong elsewhere.

Reply:
changed here and in ref manual.